### DISCUSSION TITLE
warning/reject nft mint if hashes don't match uri content

### DISCUSSION THREAD

**Discussion by Anarkoic** [Chia-Network/chia-blockchain#19048] [Ideas]
with override option?

perhaps low priority and not handling all protocol lookups, but a thought

**Comment by BrandtH22** [comment]
Hey @Anarkoic , since there is no onchain restriction for minting with a non-matching hash (and there are use cases where one would mint when the URI does not yet have content) a feature like this has not been added to the reference client. 

I will mark this as an enhancement request but I imagine this kind of feature would be best provided by one of the minting tools available that provide a more user-centric experience for minting 🤔

**Reply by Anarkoic** [reply]
excellent, agreed! thank you
